[DOCS-12674] Update RUM alerting guide (#33053)
DataDog/documentation: content/en/real_user_monitoring/guide/alerting-with-rum.md

Learn more about [creating RUM Monitors][4].

## Choosing between monitoring metrics and events

Choose what to monitor based on your use case:

- **Full traffic metrics** - Best for monitoring availability, loading times, error rates, and user experience trends. For RUM without Limits™ customers, metric-based monitors provide full unsampled traffic visibility (before retention filters apply), 15-month retention, and advanced capabilities like anomaly detection and forecasts.
- **Retained events** - Best for detecting critical, low-frequency issues that require full event context. Use event-based monitors when you need to alert on indexed data or specific attributes not available as metrics. Configure retention filters to ensure the relevant data is captured.

  To create an event-based monitor, you can export search queries from the [RUM Explorer][5]. Use any facets that RUM collects, including [custom facets and measures][6], and the `measure by` field for view-related counts like load time and error count.

## Export queries from the RUM homepage

> You can export existing queries from the [RUM homepage][12] to create a monitor with all the query and context preserved. For customers on [RUM without Limits™][1], the queries powering those widgets are based on the [out-of-the-box metrics][13]. For customers on the legacy model, they are based on [events][14].

Reviewer: The phrase "create a monitor with all the query and context preserved" reads incorrectly. Easiest fix from a grammatical standpoint would be "create a monitor with all queries and context preserved", but i'm not sure if it's factually correct.

## Overview

> Real User Monitoring (RUM) lets you create alerts about atypical behavior in your applications. With [RUM without Limits][1], use metric-based monitors to alert on your full unsampled traffic with 15-month retention. Metric-based monitors support advanced alerting conditions including anomaly detection, outlier detection, and forecasts.

Reviewer 2: ```suggestion
Real User Monitoring (RUM) lets you create alerts about atypical behavior in your applications. With [RUM without Limits][1], use metric-based monitors to alert on your full unsampled traffic with 15-month retention. Metric-based monitors support advanced alerting conditions such as anomaly detection
```
Though the other two can be achieved, our RUM template does not support other cases than anomaly at the moment so I would not emphasise this too much

## Create a RUM monitor

To create a RUM monitor in Datadog, first navigate to [Monitors > New Monitor > Real User Monitoring][2].

From there, you can click **New Monitor**, then:

- **[Start with a template][3]**: Datadog provides several pre-built templates for common RUM monitoring scenarios like error rates, performance vitals, or availability checks. Browse the [full template gallery][3] to get started.
- **Build a custom monitor**: Choose from out-of-the-box metrics or custom metrics, then scope to your application, specific pages, or views.

> Learn more about [creating RUM Monitors][4].

Reviewer 2: Should we also update [/monitors/types/real_user_monitoring](https://docs.datadoghq.com/monitors/types/real_user_monitoring/#create-a-rum-monitor)? It's outdated too

- **Visibility into your full traffic**: Metrics are computed over your full, unsampled traffic before retention filters apply to provide broader visibility into application health
- **Reduce the risk of missing issues**: Event-based monitors depend on retention filters and can miss issues if relevant events aren't indexed
- **Advanced capabilities**: Metric-based monitors support anomaly detection, outlier detection, and forecasts
- **15-month retention**: Metrics provide long-term visibility into trends and patterns

Datadog recommends using **event-based monitors** when you need to alert on data that is indexed, such as specific events or attributes that are not available in metric form. When using event-based monitors, configure retention filters to ensure meaningful data is available.

To create an event-based monitor, you can export search queries from the [RUM Explorer][5]. Use any facets that RUM collects, including [custom facets and measures][6], and the `measure by` field for view-related counts like load time and error count.

## Export queries from the RUM homepage

> For event-based monitoring use cases, you can export existing queries from the [RUM homepage][12] to create a monitor with all the query and context preserved.

Reviewer 2: ```suggestion
You can export existing queries from the [RUM homepage][12] to create a monitor with all the query and context preserved. For customers on [RUM without Limits™][1], the queries powering up those widgets are based on the [out-of-the-box metrics][13]. For customers on the legacy model, they are based on [events][14].
```

{{< img src="real_user_monitoring/guide/alerting-with-rum/traffic-anomaly.png" alt="RUM monitor query showing count of sessions metric summed by application name, with anomaly alert conditions configured to trigger when values deviate from expected patterns" style="width:100%;" >}}

## Further reading

{{< partial name="whats-next/whats-next.html" >}}

[1]: /real_user_monitoring/rum_without_limits
[2]: https://app.datadoghq.com/monitors/create/rum
[3]: https://app.datadoghq.com/monitors/templates?q=real%20user%20monitoring&origination=installed&p=1
[4]: /monitors/types/real_user_monitoring/#create-a-rum-monitor
[5]: https://app.datadoghq.com/rum/explorer
[6]: /real_user_monitoring/guide/send-rum-custom-actions/#create-facets-and-measures-on-attributes
[7]: /real_user_monitoring/explorer/export/
[8]: /monitors/notify/
[9]: /real_user_monitoring/application_monitoring/browser/advanced_configuration/?tab=npm#global-context
[10]: /real_user_monitoring/application_monitoring/browser/monitoring_page_performance/#all-performance-metrics
[11]: /real_user_monitoring/android/mobile_vitals/

> [12]: https://app.datadoghq.com/rum/performance-monitoring

Reviewer 2: ```suggestion
[12]: https://app.datadoghq.com/rum/performance-monitoring
[13]: /real_user_monitoring/rum_without_limits/metrics
[14]: /real_user_monitoring/explorer/events/
```
